graylog2-server/src/test/java/org/graylog2/indexer/retention/strategies/AbstractIndexCountBasedRetentionStrategyTest.java

```diff
@@ -81,7 +82,7 @@ public void setUp() throws Exception {
             }
 
             @Override
-            protected void retain(String indexName, IndexSet indexSet) {
+            protected void retain(List indexName, IndexSet indexSet) {
```

Comment:
Please use a generic type here. IntelliJ also should've marked this as a warning

Reply:
`List<String> indexNames`